lib/EnsEMBL/REST/Model/Registry.pm

```diff
@@ -70,6 +70,45 @@ has 'no_version_check' => ( is => 'ro', isa => 'Bool' );
 
 # Preload settings
 has 'preload' => ( is => 'ro', isa => 'Bool', default => 1 );
+has 'eqtl_dbs' => (is => 'ro', isa => 'HashRef[Str]');
+
+has '_eqtl_adaptors' => ( is => 'ro', isa => 'HashRef', lazy => 1, default => sub {
+
+  my ($self) = @_;
+  my $eqtl_adaptors = {};
+
+  return {} if(! $self->eqtl_dbs);
+  # load if not loaded
+  Catalyst::Utils::ensure_class_loaded('Bio::EnsEMBL::HDF5::EQTLAdaptor');
```

Comment:
If this is an optional bit of code then we'll have to make this include handle errors. More details from http://search.cpan.org/dist/Catalyst-Runtime/lib/Catalyst/Utils.pm#ensure_class_loaded($class_name,_\%opts)